test/services/RequestDetailsServiceSpec.scala

```diff
@@ -48,16 +48,16 @@ class RequestDetailsServiceSpec extends BaseSpec {
       }
     }
 
-    "return responseInvalidPayload" when {
-      "the dates are not parsable" in {
-        val requestDetails = createRequestDetails("non parsable", "date")
-
-        SUT.validateDates(requestDetails) mustBe Left(Constants.responseInvalidPayload)
-      }
-    }
+//    "return responseInvalidPayload" when {
+//      "the dates are not parsable" in {
+//        val requestDetails = createRequestDetails("202-04-1", "122-01-01")
+//
+//        SUT.validateDates(requestDetails) mustBe Left(Constants.responseInvalidPayload)
+//      }
+//    }
```

Comment:
Forgot something?

Reply:
Removed as validation moved to the case class